pkg/crc/services/dns/dns_darwin.go

```diff
@@ -123,37 +125,37 @@ func waitForNetwork(hostResolv *network.ResolvFileValues) error {
 	return fmt.Errorf("Host is not connected to internet.")
 }
 
-// updateResolvConfFile updates the host's /etc/resolv.conf file with Instance IP.
-func updateResolvConfFile(instanceIP string, resolvConfFile string) (bool, error) {
```

Comment:
The 'bool' return value does not seem to be needed as it's the same as (error == nil)

Reply:
@cfergeau it is required by calling function, where it is used as part of the `result` struct. or you mean just return `error == nil` and it will evaluate as bool ?

Reply:
The calling function can generate the 'success' value itself, for example:
```
if err != nil {
	result.Success = false
	return *result, err
}
```

Reply:
@cfergeau then I have to change it to more places, since all the other functions signature is same which called from the calling function.

Reply:
ah this is indeed a common pattern in this file :-/

Reply:
@cfergeau I changed all the functions signature as of now, don't want to put another commit message for this refactor.